### PR TITLE
include items json in csv export

### LINKED ISSUE #2312: [Feature request] save title or ID of items sold in transaction history
**Is your feature request related to a problem? Please describe.**
Merchants that use the tpos extension want/need an overview of the items sold in a transaction (for management and accounting)

**Describe the solution you'd like**
Add the items (title or ID) sold in a transaction to the transaction history grid and on the export to csv button.

**Describe alternatives you've considered**
none

### PR DESCRIPTION
**Pull request by talvasconcelos**
Closes #2312

### CODE REVIEW

File: lnbits/static/js/wallet.js [hunk 1]
```diff
@@ -223,6 +223,12 @@ new Vue({
             align: 'right',
             label: 'Fiat Amount',
             field: row => row.extra.wallet_fiat_amount
+          },
+          {
+            name: 'items',
+            align: 'right',
+            label: 'Items',
+            field: row => (row.tag === 'tpos' && row.extra.details) || null
```
Comment (prusnak) [inline lnbits/static/js/wallet.js:231]: This is layering violation but I guess I agree with the quick fix like this.

Maybe there should be a way how an extension can tell LNbits which extra fields should be added?

Comment (motorina0) [reply lnbits/static/js/wallet.js:231]: `lnbits core` should not know about a particular extension.
Suggestion: make the `Export` button a `<q-btn-dropdown>` with two options:
  - Basic Export
  - Detailed Export 

It defaults to `Basic Export`. Mock UI:

image

Comment (talvasconcelos) [reply lnbits/static/js/wallet.js:231]: so but then, the detailed is not only for tpos, but it just packs all the info in the 'extra', is that it? if there's something there? this hack is just to include the `details` property that only exists, so far, in tpos

Comment (arcbtc) [reply lnbits/static/js/wallet.js:231]: I'm with Vlad, lets get this merged, and make something more sophisticated later.

**Review by arcbtc — APPROVED** [review]

**Pull request closed without merging**